dvc/commands/data.py

```diff
@@ -0,0 +1,178 @@
+import argparse
+import logging
+from typing import TYPE_CHECKING
+
+from funcy import compact, log_durations
+
+from dvc.cli.command import CmdBase
+from dvc.cli.utils import append_doc_link, fix_subparsers
+from dvc.ui import ui
+
+if TYPE_CHECKING:
+    from dvc.repo.data import Status as DataStatus
+
+
+logger = logging.getLogger(__name__)
+
+
+class CmdDataStatus(CmdBase):
+    COLORS = {
+        "not_in_cache": "red",
+        "committed": "green",
+        "uncommitted": "yellow",
+        "untracked": "cyan",
+    }
+    LABELS = {
+        "not_in_cache": "Not in cache",
+        "committed": "DVC committed changes",
+        "uncommitted": "DVC uncommitted changes",
+        "untracked": "Untracked files",
+        "unchanged": "DVC unchanged files",
```

Comment:
One more.

```suggestion
    LABELS = {
        "not_in_cache": "Not in cache",
        "committed": "Committed changes",
        "uncommitted": "Uncommitted changes",
        "untracked": "Untracked files",
        "unchanged": "Unchanged files",
```

Reason: "DVC" seems unnecessary here (the command is `dvc ...` so it's assumed). And the wording can result confusing, e.g. do we mean "`dvc commit`-ed changes"? or "`git commit`-ed changes to DVC metafiles"?

Reply:
I think DVC was introduced to be extra explicit about Git vs DVC ... otherwise committed might confuse people

Reply:
I think it's clear what we're in a DVC context when using the `dvc data status` command. No strong opinion on keeping "DVC" but the wording could be clearer e.g. "Committed to DVC" and "Not committed to DVC".

Reply:
might be ... but this command takes both - Git and DVC concepts (compares workspace to HEAD). Again, it's fine to keep it super explicit here, let's not optimize here for now